apps/src/maze/api.js

```diff
@@ -254,6 +254,50 @@ exports.loopHighlight = API_FUNCTION(function (id) {
 });
 
 
-for (var functionName in Bee.api) {
-  exports[functionName] = API_FUNCTION(Bee.api[functionName]);
-}
+
+/**
+ * Bee related API functions. If better modularized, we could potentially
+ * separate these out, but as things stand right now they will be loaded
+ * whether or not we're a Bee level
+ */
+exports.getNectar = API_FUNCTION(function(id) {
```

Comment:
know this is moved code, but linter shouldn't yell about the `function(` versus `function (` right?